lib/services/monitor/lib/models/index.d.ts

```diff
@@ -196,30 +1610,30 @@ export interface HttpRequestInfo {
  * @member {string} [tenantId] the Azure tenant Id
  */
 export interface EventData {
-  authorization?: SenderAuthorization;
-  claims?: { [propertyName: string]: string };
-  caller?: string;
-  description?: string;
-  id?: string;
-  eventDataId?: string;
-  correlationId?: string;
-  eventName?: LocalizableString;
-  category?: LocalizableString;
-  httpRequest?: HttpRequestInfo;
-  level: string;
-  resourceGroupName?: string;
-  resourceProviderName?: LocalizableString;
-  resourceId?: string;
-  resourceType?: LocalizableString;
-  operationId?: string;
-  operationName?: LocalizableString;
-  properties?: { [propertyName: string]: string };
-  status?: LocalizableString;
-  subStatus?: LocalizableString;
-  eventTimestamp: Date;
-  submissionTimestamp: Date;
-  subscriptionId?: string;
-  tenantId?: string;
+  readonly authorization?: SenderAuthorization;
+  readonly claims?: { [propertyName: string]: string };
+  readonly caller?: string;
```

Comment:
Marking properties as readonly is a breaking change. Requires major version increment.